containers/docker-compose.yaml

```diff
@@ -49,7 +52,7 @@ services:
       - GDS_BACKUP_KEEP=1
       - GOOGLE_APPLICATION_CREDENTIALS
       - GOOGLE_PROJECT_NAME
-      - GDS_SECRETS_TESTING
+      - GDS_SECRETS_TESTING=true
```

Comment:
`GDS_SECRETS_TESTING` uses the mock secrets manager rather than google secret manager, which I think is what we want when we're running things locally. One of `GOOGLE_APPLICATION_CREDENTIALS` or `GDS_SECRETS_TESTING` needs to be set. Perhaps it is better to just leave this to the environment though; I'll change it back.